lib/faraday.rb

```diff
@@ -169,6 +149,4 @@ def method_missing(name, *args, &block)
   self.root_path = File.expand_path __dir__
   self.lib_path = File.expand_path 'faraday', __dir__
   self.default_adapter = :net_http
-
-  require_lib 'autoload' unless ENV['FARADAY_NO_AUTOLOAD']
```

Comment:
Is this ENV var used anywhere in the code or CI or anything? Was it just an undocumented extra?

Reply:
I searched for it but couldn't find any reference